spytial/dataclass_widget.py

```diff
@@ -109,13 +121,62 @@ def _setup_widget(self):
                                 console.error('Jupyter kernel not available');
                             }}
                         }}
+                        
+                        // Handle value request response from iframe
+                        else if (event.data && event.data.type === 'spytial-value-response' && event.data.widgetId === '{self._widget_id}') {{
+                            console.log('Received value response:', event.data);
+                            
+                            // Execute Python to set response data
+                            var code = `
+# Handle value request response
```

Comment:
This always times out. How are you even calling dataclassbuilder functions (Python) from JS?